components/CodeBlock.tsx

```diff
@@ -1,39 +1,94 @@
+import * as Components from '@atom-learning/components'
 import Highlight, { defaultProps } from 'prism-react-renderer'
 import theme from 'prism-react-renderer/themes/nightOwl'
 import * as React from 'react'
+import { LiveEditor, LiveError, LivePreview, LiveProvider } from 'react-live'
 
 import { styled } from '../stitches.config'
 
+type CodeBlockProps = {
+  children: string
+  live?: boolean
```

Comment:
Could this be `isLive`?

Reply:
This is coming from the `react-live` component, or at least it's often used as the signifier to render a live editor/preview. The markdown snippet would be `''' jsx isLive` as opposed to `''' jsx live`. I am leaning towards keeping it as is because: a) it's a common convention I've seen with `react-live`, and b) it's also used in markdown, which doesn't necessarily require the same strict naming conventions.

Thoughts?

Reply:
Huh, interesting! Yeah, I'm certainly happy for that to stay then. Thanks for the explanation.

I've thought more about this since making that first comment and there are definitely cases where it _feels nice_ to pass a boolean as a prop with the word that describes it, rather than the traditional boolean-looking name, and I would say this is one. If they can be categorised well enough then I think that would be a good candidate for something to be documented in the React-specific handbook we were chatting about last week. I'll have a think!